src/block.rs

```diff
@@ -73,31 +73,31 @@ pub struct Block<T> {
 }
 
 impl<T: Serialize + Clone> Block<T> {
-    /// A new `Block` requires a valid vote and the `PeerId` of the peer who sent us this. For
+    /// A new `Block` requires a valid vote and the `PublicInfo` of the peer who sent us this. For
     /// this reason the `Vote` will require a `DirectMessage` from a peer to us.
-    pub fn new(vote: &Vote<T>, peer_id: &PeerId) -> Result<Block<T>, RoutingError> {
+    pub fn new(vote: &Vote<T>, peer_info: &PublicInfo) -> Result<Block<T>, RoutingError> {
```

Comment:
Most if not all instances of `peer` in this PR should be changed to `node`.

Reply:
Think this is kind of depends on whether your PR 1533 (task 2467) got merged or this one  :)

Reply:
Yeah - I guess I should have talked to you about that one first.  I left all instances of `peer_id` alone in my PR to try to avoid needless conflicts! 😄